src/main/scala/cromwell/engine/backend/CromwellBackend.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.engine.backend
+
+import akka.actor.ActorSystem
+import org.slf4j.LoggerFactory
+import Backend.BackendyString
+
+/**
+  * Provides a global singleton access to the instantiated backend. This isn't bulletproof and can lead to
+  * exceptions being thrown but the way the system utilizes it it shouldn't be a problem (famous last words).
+  * So far the only real oddities I've seen are from the unit tests when they were trying to go straight to the
+  * interior of the system.
+  *
+  * Also we expect nearly everything to change about Backend in the next couple of months due to Intel's
```

Comment:
@scottfrazer @geoffjentry Just a word regarding this whole discussion: The point of the changes that we were intending to do currently is that if anyone who wants to add a new Backend can do so without changing the Parser/Binding (or in that case Engine as well) code in Cromwell. Just add a jar for your implementation (as a 'plugin'), make a configuration change to change the Backend class to instantiate, and it should load the new backend, as opposed to the current way of things where there is a change involved in all those layers to add a backend. 

I think we discussed this idea previously, but didn't dive into the technical details, which we can do sometime right now if we are not too restricted from our end of year deadlines and commitments in our respective offices :)